whois-nrtm4/src/test/java/net/ripe/db/nrtm4/SnapshotFileGeneratorIntegrationTest.java

```diff
@@ -45,9 +45,9 @@ public void initial_snapshot_file_is_generated_and_written_to_disk() throws IOEx
         loadScripts(whoisTemplate, "nrtm_sample_sm.sql");
         final String sessionID;
```

Comment:
Can we add a testcase or test dataq for mutiple sources not just TEST source?

Reply:
Can we try to use DatabaseHelper in setup class, if we are not dealing with lot of objects? Its easier to navigate and no need to add manual entries in serials and history tables

Reply:
It's ok if you prefer sql dump and if we have been using it other places. Its just for me its easier to read other way but its personal choice :)